man/git-alias.md

```diff
@@ -6,13 +6,23 @@ git-alias(1) -- Define, search and show aliases
 `git-alias`  
 `git-alias` &lt;search-pattern&gt;  
 `git-alias` &lt;alias-name&gt; &lt;command&gt;  
+`git-alias` --global &lt;alias-name&gt; &lt;command&gt;  
+`git-alias` --local &lt;alias-name&gt; &lt;command&gt;  
 
 ## DESCRIPTION
 
   List all aliases, show one alias, or set one (global) alias.
 
 ## OPTIONS
 
+  &lt;--global&gt;
+
+  write and read the config from ~/.gitconfig file only.
```

Comment:
Would be better to focus on the behavior of the cmd, instead of the implementation detail.

Reply:
Got this idea, I will refactor the description.